tests/test_e2e_70_kytos_stats.py

```diff
@@ -62,16 +62,16 @@ def test_010_table_stats(self):
         response = requests.get(api_url)
         assert response.status_code == 200, response.text
         data = response.json()
-        assert len(data) == 1
-        assert '00:00:00:00:00:00:00:01' in data
+        assert len(data) == 1, str(data)
+        assert '00:00:00:00:00:00:00:01' in data, str(data)
```

Comment:
Typically we only add extra context to the assert messages when pytest doesn't add much info given a boolean comparison like `len(data) == 1`. For instance, in a string `in` assertion it typically includes, even in older version of pytest:

```
E       AssertionError: assert 'a' in 'some'
```

The extra content won't cause harm though, but if we don't have too then we're just adding unnecessary information, can you simulate an assertion error for this equivalent case and see if it's really needed?